--- src/hotspot/cpu/aarch64/icache_aarch64.hpp	2021-03-19 21:07:28.000000000 +0100
+++ src/hotspot/cpu/aarch64/icache_aarch64.hpp	2021-03-19 21:08:10.000000000 +0100
@@ -30,6 +30,10 @@
 // modifies code, part of the processor instruction cache potentially
 // has to be flushed.

+extern "C" {
+  void __clear_cache(char *beg, char *end);
+}
+
 class ICache : public AbstractICache {
  public:
   static void initialize();
